src/Layout.js

```diff
@@ -1506,13 +1541,13 @@ var computeLayout = (function() {
 
       // First, try to use the layout cache.
       if (layout.cachedLayout &&
-          canUseCachedMeasurement(availableWidth, availableHeight, marginAxisRow, marginAxisColumn,
+          canUseCachedMeasurement(node.isTextNode, availableWidth, availableHeight, marginAxisRow, marginAxisColumn,
```

Comment:
Slightly mixed about calling this "isTextNode" instead of something like "isHeightForWidth". I don't feel strongly about this though. Maybe text is a more intuitive than the concept of height-for-width layouts.